core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/OpenWhiskEvents.scala

```diff
@@ -29,11 +29,14 @@ import org.apache.kafka.common.serialization.StringDeserializer
 import pureconfig._
 import pureconfig.generic.auto._
 
+import scala.concurrent.duration.FiniteDuration
 import scala.concurrent.{ExecutionContext, Future}
 
 object OpenWhiskEvents extends SLF4JLogging {
 
-  case class MetricConfig(port: Int, enableKamon: Boolean, ignoredNamespaces: Set[String])
+  case class MetricConfig(port: Int, enableKamon: Boolean, ignoredNamespaces: Set[String], retry: RetryConfig)
+
+  case class RetryConfig(minBackoff: FiniteDuration, maxBackoff: FiniteDuration, randomFactor: Double, maxRestarts: Int)
```

Comment:
@chetanmeh Thank you for your review. I've used the same Configuration class here.   
https://github.com/apache/openwhisk/pull/4632/files#diff-3753d510c1485477c63054dde55b759dR39

Reply:
@chetanmeh I'm sorry that I forgot to tell you that I referenced your code 😅